lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/Jordan_Eldridge/AnimalTest.java

```diff
@@ -0,0 +1,40 @@
+package com.codedifferently.lesson7.Jordan_Eldridge;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.codedifferently.lesson7.Jordan_Eldridge.Animal.AnimalType;
+import org.junit.jupiter.api.Test;
+
+public class AnimalTest {
+
+  @Test
+  public void testCanBreathe() {
```

Comment:
The first four tests are really just testing the same thing (that the animal has been constructed correctly once the constructor is called). Merge into one that just asserts all the properties are set correctly.